examples/tof-viewer/tof-viewer_config.json

```diff
@@ -1,8 +1,8 @@
 {
     "VAUX_POWER_VOLTAGE": "18",
-    "skip_network_cameras": "off",
```

Comment:
Remove the changes from this file, keep the master version.

Reply:
fixed